torquedata/torquedata/__init__.py

```diff
@@ -61,6 +62,15 @@ def cull_invalid_objects(group, sheet_name, wiki_key):
     else:
         return list(data[sheet_name].values())
 
+def update_row_with_edits(row, sheet_name, key):
+    row_edits = edits[sheet_name][key]
+
+    for k in row:
+        if len(row_edits[k]['edits']) > 0:
+            row[k] = row_edits[k]['edits'][-1]['new_value']
```

Comment:
It doesn't really matter, since we're storing a timestamp with each edit, but doesn't this place the newest edit at the end instead of at the beginning of the list of edits?

Reply:
That is correct. As you can see here, [we append edits](https://github.com/OpenTechStrategies/torque/pull/41/files#diff-36d45131c2e7ca6477378559776f7226R142)